site/content/docs/monitoring/false-positives.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Understanding and managing false positives
+weight: 10
+menu:
+  platform:
+    parent: "Monitoring"
+---
+
+At Checkly, we provide a reliable and performant solution for running synthetic monitoring and API checks across 22 regions. Our infrastructure is designed to be robust, but sometimes, unforeseen issues like server or network problems can crop up—whether on our side or yours. Often, these issues show up as isolated incidents that resolve on their own. However, in the worst cases, they can cause false positives in your check results, where Checkly might report that your site or server is slow or unavailable, even though you know it's up and running.
```

Comment:
I feel like this is written too much as "Checkly has a problem". This is more like inherently to how public internet works. I'd rewrite it in a way explaining that inbetween Checkly and your check target, there are many points which neither us nor the user can control, so temporary problems, rerouting etc can occur and is TOTALLY normal.

Reply:
Agreed, will rewrite!